test/JDBC/expected/BABEL-3395-vu-verify.out

```diff
@@ -0,0 +1,811 @@
+GO
+~~START~~
+varchar
+12345.1
+~~END~~
+
+~~START~~
+varchar
+12345.1
+~~END~~
+
+
+DECLARE @bigint bigint = 2;
+SELECT UPPER(@bigint)
+SELECT LOWER(@bigint)
+GO
+~~START~~
+varchar
+2
+~~END~~
+
+~~START~~
+varchar
+2
+~~END~~
+
+
+DECLARE @int int = 2;
+SELECT UPPER(@int)
+SELECT LOWER(@int)
+GO
+~~START~~
+varchar
+2
+~~END~~
+
+~~START~~
+varchar
+2
+~~END~~
+
+
+DECLARE @smallint smallint = 2;
+SELECT UPPER(@smallint)
+SELECT LOWER(@smallint)
+GO
+~~START~~
+varchar
+2
+~~END~~
+
+~~START~~
+varchar
+2
+~~END~~
+
+
+DECLARE @tinyint tinyint = 2;
+SELECT UPPER(@tinyint)
+SELECT LOWER(@tinyint)
+GO
+~~START~~
+varchar
+2
+~~END~~
+
+~~START~~
+varchar
+2
+~~END~~
+
+
+DECLARE @money money = 2;
+SELECT UPPER(@money)
+SELECT LOWER(@money)
+GO
+~~START~~
+varchar
+2.0000
+~~END~~
+
+~~START~~
+varchar
+2.0000
+~~END~~
+
+
+DECLARE @smallmoney smallmoney = 2;
+SELECT UPPER(@smallmoney)
+SELECT LOWER(@smallmoney)
+GO
+~~START~~
+varchar
+2.0000
+~~END~~
+
+~~START~~
+varchar
+2.0000
+~~END~~
+
+
+DECLARE @bit bit = 1;
+SELECT UPPER(@bit)
+SELECT LOWER(@bit)
+GO
+~~START~~
+varchar
+1
+~~END~~
+
+~~START~~
+varchar
+1
+~~END~~
+
+
+DECLARE @myid uniqueidentifier = CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS uniqueidentifier)
+SELECT UPPER(@myid)
+SELECT LOWER(@myid)
+GO
+~~START~~
+varchar
+6F9619FF-8B86-D011-B42D-00C04FC964FF
+~~END~~
+
+~~START~~
+varchar
+6f9619ff-8b86-d011-b42d-00c04fc964ff
+~~END~~
+
+
+DECLARE @myid sql_variant = CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS sql_variant)
+SELECT UPPER(@myid)
+SELECT LOWER(@myid)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Argument data type sql_variant is invalid for argument 1 of upper function.)~~
+
+
+DECLARE @myid xml = CAST ('<body/>' AS xml)
+SELECT UPPER(@myid)
+SELECT LOWER(@myid)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Argument data type xml is invalid for argument 1 of upper function.)~~
+
+
+DECLARE @myid geometry = geometry::STGeomFromText('POINT (1 2)', 0)
+SELECT UPPER(@myid)
+SELECT LOWER(@myid)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Argument data type geometry is invalid for argument 1 of upper function.)~~
+
+
+DECLARE @myid sql_variant = CAST ('6F9619FF-8B86-D011-B42D-00C04FC964FF' AS sql_variant)
+SELECT UPPER(CAST (@myid AS VARCHAR))
+SELECT LOWER(CAST (@myid AS VARCHAR))
+GO
+~~START~~
+varchar
+6F9619FF-8B86-D011-B42D-00C04F
+~~END~~
+
+~~START~~
+varchar
+6f9619ff-8b86-d011-b42d-00c04f
+~~END~~
+
+
+DECLARE @myid xml = CAST ('<body/>' AS xml)
+SELECT UPPER(CAST (@myid AS VARCHAR))
+SELECT LOWER(CAST (@myid AS VARCHAR))
+GO
+~~START~~
+varchar
+<BODY/>
+~~END~~
+
+~~START~~
+varchar
+<body/>
+~~END~~
+
+
+DECLARE @myid geometry = geometry::STGeomFromText('POINT (1 2)', 0)
+SELECT UPPER(CAST (@myid AS VARCHAR))
+SELECT LOWER(CAST (@myid AS VARCHAR))
+GO
+~~START~~
+varchar
+POINT(1 2)
+~~END~~
+
+~~START~~
+varchar
+point(1 2)
+~~END~~
+
+
+Select UPPER(a), LOWER(a) from upper_lower_text
+GO
+~~START~~
+text#!#text
```

Comment:
`UPPER` and `LOWER` function should always return `varchar` or `nvarchar`.

Reply:
Done, Added Wrapper